### PR TITLE
profiles: cover uploading state

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes TEAM2-355
Figma link (if any):

## What changed (plus any additional context for devs)
added a loading/uploading state (usually pretty quick but fixes the lack of user feedback )

## Screen recordings / screenshots

https://user-images.githubusercontent.com/29204161/183518995-779e6527-01ec-4409-b20d-0751c9599a50.mov

## What to test

upload cover photos 

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/components/ens-registration/RegistrationCover/RegistrationCover.tsx [hunk 1]
```diff
@@ -102,6 +107,8 @@ const RegistrationCover = ({
     },
     [onBlurField, setCoverMetadata]
   );
+  const [isUploading, setIsUploading] = useState(false);
+  const [isLoadingImage, setIsLoadingImage] = useState(false);
```
Comment (estebanmino) [inline src/components/ens-registration/RegistrationCover/RegistrationCover.tsx:111]: i'm not following why we need two states `isUploading` and `isLoadingImage` if they're both being used to show `Uploading` on the ui https://github.com/rainbow-me/rainbow/pull/3963/files#diff-28a65281fd94c155c98670d0b2c1602d01b2fe3a7509da60294d854aee700bfeR189

Comment (estebanmino) [reply src/components/ens-registration/RegistrationCover/RegistrationCover.tsx:111]: names make sense but wondering if we could just use one `isImageReady` or something like that

Comment (jxom) [reply src/components/ens-registration/RegistrationCover/RegistrationCover.tsx:111]: do we even need this state? `useSelectImageMenu` also returns `isUploading`.

**Comment by jxom** [comment]
We previously had uploading UI – I wonder if we can just bring back the UI from here:

Cover: https://github.com/rainbow-me/rainbow/pull/3047/files#diff-28a65281fd94c155c98670d0b2c1602d01b2fe3a7509da60294d854aee700bfeL84-L103

Avatar: https://github.com/rainbow-me/rainbow/pull/3047/files#diff-979c9a5a3bd5e6c060d240887a5c189b4b70b4198d86222ce45e89c2e4a44478L119-L139

**Review by jxom — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]

**Pull request merged**